modules/core/src/v2/baseCoin.ts

```diff
@@ -15,6 +15,14 @@ const Keychains = require('./keychains');
 const PendingApprovals = require('./pendingApprovals');
 import { Enterprises } from './enterprises';
 
+/**
+ * Basic recipient information
+ */
+export interface Recipient {
```

Comment:
This would be more appropriate in types.ts instead of baseCoin.

Reply:
Moved.